src/System.Data.Common/src/System/Data/Common/DbConnection.cs

```diff
@@ -35,6 +35,8 @@ protected DbConnection() : base()
         /// </summary>
         protected virtual DbProviderFactory DbProviderFactory => null;
 
+        internal DbProviderFactory ProviderFactory=>DbProviderFactory;
```

Comment:
Also, @ericstj, is illink going to remove this?  @FransBouma would need to do something to prevent that, right?